templates/ts-apollo-fullstack/server/src/index.ts

```diff
@@ -1,30 +1,25 @@
+import http from 'http';
 import cors from 'cors';
 import express from 'express';
-import http from 'http';
-
+import dotenv from 'dotenv'
 import { createApolloServer } from './graphql';
 
-async function start() {
-  const app = express();
+dotenv.config()
```

Comment:
This should be done at very very top of the imports. If not it can cause bugs where environment will not be set.

Reply:
Added now in https://github.com/aerogear/graphback/pull/1069/commits/9f8acb4cadc0a42cf77178ce7fd203e5eebd2448